ApiFirst/DomainModels/Models/CustomerBankInfo.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace DomainModels.Models
+{
+    public class CustomerBankInfo : BaseModel
+    {
+        public CustomerBankInfo(long customerId)
+        {
+            CustomerId = customerId;
+        }
+
+        public CustomerBankInfo(decimal accountBalance, long customerId, Customer customer)
+        {
+            AccountBalance = accountBalance;
+            CustomerId = customerId;
+            Customer = customer;
+        }
+
+        public decimal AccountBalance { get; set; } = 0.0m;
```

Comment:
```suggestion
        public decimal AccountBalance { get; set; }
```
Por padrão o valor inicial já é 0